src/core_modules/capture-core/components/DataEntry/withBrowserBackWarning.js

```diff
@@ -68,11 +68,11 @@ const getEventListener = (InnerComponent: React.ComponentType<any>) =>
                         {...passOnProps}
                     />
                     <ConfirmDialog
-                        header={i18n.t('Unsaved changes')}
-                        text={i18n.t('Leaving this page will discard the changes you made to this event.')}
-                        confirmText={i18n.t('Yes, discard')}
+                        header={i18n.t('Discard unsaved changes?')}
```

Comment:
Hey,
Can we reuse the `header`, `text`, `destructiveText` and `cancelText` const values from [ConfirmDialog.constants.js](https://github.com/dhis2/capture-app/blob/2a9930553b4567128db84a43fe9104132e16bbc0/src/core_modules/capture-core/components/Dialogs/ConfirmDialog.constants.js)?
Thanks!